js/src/components/modals/KeyphrasesTable.js

```diff
@@ -0,0 +1,44 @@
+/* External dependencies */
+import React from "react";
+import PropTypes from "prop-types";
+import { Fragment } from "@wordpress/element";
+
+/**
+ * The Related Keyphrases table component.
+ */
+class KeyphrasesTable extends React.Component {
+	/**
+	 * Constructs the Related Keyphrases table.
```

Comment:
Missing `@summary`.

Reply:
This will be done in a separate issue on Jira